vars/testRpm.groovy

```diff
@@ -0,0 +1,80 @@
+// vars/testRpm.groovy
+
+  /**
+   * testRpm step method
+   *
+   * @param config Map of parameters passed
+   *
+   * config['context']      Context name for SCM to identify the specific
+   *                        stage to update status for.
+   *                        Default is 'test/' + env.STAGE_NAME.
+   *
+   *  Important:
+   *     The SCM status checking for passing may expect a specific name.
+   *
+   *     Matrix stages must override this setting to include matrix axes
+   *     names to ensure a unique name is generated.
+   *
+   *     Or the default name has to be changed in a way that is compatible
+   *     with a future Matrix implementation.
+   *
+   * config['daos_pkg_version']  Version of DAOS package.  Required.
+   *
+   * config['description']       Description to report for SCM status.
+   *                             Default env.STAGE_NAME.
+   *
+   * config['failure_artifacts'] Failure aritfifacts to return.
+   *                             Default env.STAGE_NAME.
+   *
+   * config['ignore_failure']    Ignore test failures.  Default false.
+   * config['inst_repos']        Additional repositories to use.  Optional.
+   *
+   * config['inst_rpms']         Additional rpms to install.  Optional
+   *
+   * config['junit_files']       Junit files to return.  Optional.
+   *
+   * config['NODELIST']          NODELIST of nodes to run tests on.
+   *                             Default env.NODELIST
+   *
+   * config['target']            Target distribution, such as 'centos7',
+   *                             'leap15'.  Default based on parsing
+   *                             environment variables for the stage.
+   *
+   * config['test_script']       Script to build RPMs. 
+   *                             Default 'ci/rpm/test_daos.sh'.
+   *
```

Comment:
`config['profile']` is missing per https://github.com/daos-stack/daos/blob/5a51a1f55473f454fc1b752c181db9cd3b725be3/Jenkinsfile#L1109

Reply:
Will be fixed in next push.